Source/SwiftLintFramework/Rules/Lint/EmptyMethodOverrideRule.swift

```diff
@@ -0,0 +1,158 @@
+import Foundation
+import SourceKittenFramework
+
+public struct EmptyMethodOverrideRule: SubstitutionCorrectableASTRule,
+ConfigurationProviderRule, AutomaticTestableRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    static let nonTriggeringExamples = [
+        """
+        override func viewDidDisappear(_ animated: Bool) {
```

Comment:
I like to think of `"""` like of opening braces. The only problem is the closing `"""` which defines if there will be spaces added to the String. Thus I use this:

```swift
    """
        override func viewDidDisappear(_ animated: Bool) {
            super.viewDidDisappear(animated)
            print("View controller did disappear")
        }
        """,
    """
        ...
        """,
```

But this doesn't bother me too much as it also looks quite clean and readable.

Reply:
I like the balance of the indentation...